### PR TITLE
chore: update docs on adding icons

### PR DESCRIPTION
**Pull request by TheSisb**
Adding documentation from the engineering side on how to self-service icon additions.

### CODE REVIEW

File: packages/paste-website/src/pages/icon-system/how-to-add-an-icon.mdx [hunk 1]
```diff
@@ -28,103 +28,126 @@ export const pageQuery = graphql`
...
+
+### Self-service instructions
+
+1. Fork the Paste repository because PRs can only be open against forks and not branches for security reasons.
+2. Clone your newly forked Paste repository:
+```
+git clone <url>
+```
+3. Navigate inside the newly cloned repository with `cd paste`
+4. Move any number of `svg` icon files into the `packages/paste-icons/svg/` folder.
```
Comment (SiTaggart) [inline packages/paste-website/src/pages/icon-system/how-to-add-an-icon.mdx]:
```suggestion
4. For each new icon you would like to create, add an `svg` icon file into the `packages/paste-icons/svg/` folder.
```

File: packages/paste-website/src/pages/icon-system/how-to-add-an-icon.mdx [hunk 2]
```diff
@@ -28,103 +28,126 @@ export const pageQuery = graphql`
...
+5. From the root Paste folder, run this automatic conversion command:
```
Comment (SiTaggart) [inline packages/paste-website/src/pages/icon-system/how-to-add-an-icon.mdx]:
```suggestion
5. From the root Paste folder, run the following command in your terminal:
```

File: packages/paste-website/src/pages/icon-system/how-to-add-an-icon.mdx [hunk 3]
```diff
@@ -28,103 +28,126 @@ export const pageQuery = graphql`
...
+```
+yarn workspace @twilio-paste/icons convert-new
+```
+6. Verify the `packages/twilio-icons/src` folder to see if your new icons were added correctly.
```
Comment (SiTaggart) [inline packages/paste-website/src/pages/icon-system/how-to-add-an-icon.mdx]:
```suggestion
6. Verify your new icons were added correctly by checking the `packages/twilio-icons/src` folder to see the newly  created source files.
```

File: packages/paste-website/src/pages/icon-system/how-to-add-an-icon.mdx [hunk 4]
```diff
@@ -28,103 +28,126 @@ export const pageQuery = graphql`
...
+7. At this point, run the following command to update our documentation with the new icons:
+```
+yarn workspace @twilio-paste/icons list-icons
+```
+8. Commit your changes and submit a new PR on Github!
```
Comment (serifluous) [inline packages/paste-website/src/pages/icon-system/how-to-add-an-icon.mdx]: On the actual page, the numbering resets after every code block.

**Pull request merged**